docs/upgrades.md

```diff
@@ -180,7 +180,8 @@ Consult the [tags list](https://github.com/mezo-org/mezod/tags) for full version
 | `v0.5.0` | 2213000 | Hard fork upgrade without chain halt | On-chain precompile versioning. New Upgrade and PriceOracle precompiles and upgrade of the existing Maintenance precompile.                                                             |
 | `v0.6.0` | 2563000 | Hard fork upgrade without chain halt | Introduce the ERC20 bridge and the BTC supply assertion.                                                                                                                                |
 | `v0.7.0` | 3078794 | Hard fork upgrade without chain halt | Fix security issues in the EVM state DB. Introduce proper reverts for precompiles. Add chain fee splitter support. Disable Cosmos transactions.                                         |
-| `v1.0.0` | 3569000 | Hard fork upgrade with chain halt    | Patch for a DoS vector in the bridge. Fix for the precompile revert mechanism                                                                                                           |
+| `v1.0.0-rc0` | 3569000 | Planned upgrade with chain halt    | Patch for a DoS vector in the bridge. Fix for the precompile revert mechanism.                                                                                                           |
+| `v1.0.0-rc1` | 3710000 | Planned upgrade with chain halt    | Patch for mixed precompile addresses.                                                                                                           |
```

Comment:
This block ETA is Apr 9, ~09:30 UTC. This is quite early for US-based validators. WDYT about pushing it around 12 UTC? This is typically the time we schedule upgrades at.

Reply:
Sure thing, that was my intention but miscalculated that [4c4b0e7](https://github.com/mezo-org/mezod/pull/438/commits/4c4b0e7d2637c922218405d0574174c0b0958318)